src/components/ReactIcon/ReactIcon.test.tsx

```diff
@@ -0,0 +1,11 @@
+import { render, screen } from '@testing-library/react';
+
+import { ReactIcon } from './ReactIcon';
+
+describe('Test AppButton component', () => {
```

Comment:
the description of `describe` block contains only component name, Icon - in this case